python/ray/data/_internal/logical/operators/read_operator.py

```diff
@@ -33,6 +34,16 @@ def __init__(
         self._concurrency = concurrency
         self._detected_parallelism = None
 
+    def dataset_export_args(self) -> Dict[str, Any]:
+        """Export the arguments into dictionary format."""
+        return {
+            "datasource": self._datasource.get_name(),
```

Comment:
nit, not blocking this PR. but I think we can also make the `Datasource.__str__` more meaningful. E.g. `ParquetDatasource(inputs="...")`